css/components/dashboards/dashboards-gallery.scss

```diff
@@ -0,0 +1,142 @@
+.c-dashboards-gallery-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  .thumbnail-list-item {
+    position: relative;
+    height: 200px;
+    border-radius: 4px;
+    background-color: $color-dark-grey;
+    background-size: cover;
+    background-position: center center;
+    outline: none;
+    padding: 0;
+    transition: all $animation-time-2 $ease-in-out-sine;
+
+    &:nth-of-type(6n) {
+      margin-right: 0;
+    }
+
+    // The input is just here for an accessibility matter
+    // we don't want it to be visually present
+    input[type="radio"] {
+      position: absolute;
+      top: 0;
+      left: -10000px;
+
+      &:focus + .content {
+        border: 1px solid $color-primary;
+      }
+    }
+
+    .content {
+      display: flex;
+      justify-content: flex-start;
+      align-items: flex-end;
+      width: 100%;
+      height: 100%;
+      font-size: $font-size-extra-big;
+      font-weight: $font-weight-bold;
+      color: $color-white;
+      border-radius: 4px;
+      cursor: pointer;
+      box-shadow: 0 2px 6px 0 rgba(0,0,0,0.1);
+      
+      .text {
+        width: 100%;
+        min-height: 75px;
+        background-color: $white;
+
+        
+        .dashboard-title {
+          color: #393F44;
+          font-family: Lato;
+          font-size: 16px;
+          font-weight: bold;
+          line-height: 20px;
+          text-align: left;
+          padding: 15px 15px 10px 15px;
```

Comment:
Please use the `$space...` CSS variables defined in `_settings.scss` for this and any other similar cases 😉 .

Reply:
Fixed here: https://github.com/resource-watch/resource-watch/pull/1185/commits/68daf5ece578aa9e34947bc6518d91f784da794a